lib/plugins/exchange/cctx/cctx.js

```diff
@@ -0,0 +1,57 @@
+var ccxt = require('ccxt')
+const coinUtils = require('../../../coin-utils')
+const _ = require('lodash/fp')
+const common = require('../../common/cctx')
+
+function trade (side, account, cryptoAtoms, fiatCode, cryptoCode, exchangeName) {
+  try {
+    const exchange = setUpExchange(account, exchangeName)
+    const symbol = common.verifyCurrencies(exchangeName, fiatCode, cryptoCode)
+    const amount = coinUtils.toUnit(cryptoAtoms, cryptoCode).toFixed(8)
+    if (exchangeName === 'itbit') {
+      return exchange.fetchOrderBook(symbol)
+        .then(orderBook => {
+          return exchange.createOrder(symbol, 'limit', side, amount, calculatePrice(side, amount, orderBook), { walletId: account.walletId })
+        })
+    }
+    return exchange.createOrder(symbol, 'market', side, amount)
+  } catch (e) {
+    return Promise.reject(e)
+  }
+}
+
+function setUpExchange (account, exchangeName) {
+  // map given credentials to cctx properties
+  if (!_.includes(exchangeName, ccxt.exchanges)) {

```

Comment:
I think you can use _.negate instead of ! but I'm not sure if it's worth the extra verbosity... confirm with Rafael
also, remove braces

Reply:
Yeah, `!` is fine here.